shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -88,13 +140,153 @@ - (UIAccessibilityTraits)accessibilityTraits {
 
 @end  // FlutterSwitchSemanticsObject
 
-@implementation FlutterCustomAccessibilityAction {
-}
+@interface FlutterScrollableSemanticsObject ()
+@property(nonatomic, strong) SemanticsObject* semanticsObject;
 @end
 
-@interface SemanticsObject ()
-/** Should only be called in conjunction with setting child/parent relationship. */
-- (void)privateSetParent:(SemanticsObject*)parent;
+@implementation FlutterScrollableSemanticsObject {
+  fml::scoped_nsobject<SemanticsObjectContainer> _container;
+}
+
+- (instancetype)initWithSemanticsObject:(SemanticsObject*)semanticsObject {
+  self = [super initWithFrame:CGRectZero];
+  if (self) {
+    self.semanticsObject = semanticsObject;
```

Comment:
You should use `_semanticsObject = [semanticsObject retain];`  You aren't suppose to reference properties through self in init methods.